x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -348,11 +349,22 @@ export const filterExceptionItems = (
 ): Array<ExceptionListItemSchema | CreateExceptionListItemSchema> => {
   return exceptions.reduce<Array<ExceptionListItemSchema | CreateExceptionListItemSchema>>(
     (acc, exception) => {
-      const entries = exception.entries.filter((t) => entry.is(t) || entriesNested.is(t));
+      const entries = exception.entries.filter((t) => {
+        const [validatedEntry] = validate(t, entry);
+        const [validatedNestedEntry] = validate(t, entriesNested);
+
+        if (validatedEntry != null || validatedNestedEntry != null) {
+          return true;
+        }
+
+        return false;
```

Comment:
```suggestion
        return validatedEntry != null || validatedNestedEntry != null;
```

Reply:
Why are we switching from `is` to `validate` here?

Reply:
I found that using `.is` only checked for the existence of the required fields, it did not actually validate those fields, so empty string was still going through as valid.